extension/src/services/pages.ts

```diff
@@ -74,6 +75,9 @@ export class PageDelegate {
     }
 
     isVideoPage() {
+        if (this.config.hash) {
```

Comment:
Now that both `hash` and `pathname` are possible I think it would make sense to treat them both as optional and test each setting if it is present. Right now `pathname` is ignored if `hash` is present.

Reply:
Makes sense, updated to include both in [df1e15f](https://github.com/killergerbah/asbplayer/pull/365/commits/df1e15ffd19897c24ab4a0c383bd707726b83fbc)